### PR TITLE
Update podspec to include bubble-wrap bundled style

### LINKED ISSUE #156: `pod install Mapzen-ios-sdk` currently doesn't bundle bubble-wrap base style

### PR DESCRIPTION
**Pull request by msmollin**
Fixes #156

### CODE REVIEW

File: Mapzen-ios-sdk.podspec [hunk 1]
```diff
@@ -23,6 +23,6 @@ Pod::Spec.new do |s|
     cs.dependency 'OnTheRoad', '~> 1.0.0-beta'
     cs.dependency 'Tangram-es', '~> 0.4'
     cs.source_files = "src/*.swift"
-    cs.resources = 'images/*.png'
+    cs.resources = [ 'images/*.png', 'tangram/*' ]
```
Comment (ecgreb) [inline Mapzen-ios-sdk.podspec]: Will this recursively include all subfolders of `tangram/*`?

Comment (msmollin) [inline Mapzen-ios-sdk.podspec]: Yes it does.

Comment (msmollin) [inline Mapzen-ios-sdk.podspec]: Ends up looking like this when imported. 
screen shot 2017-02-01 at 10 32 42 am

Comment (ecgreb) [inline Mapzen-ios-sdk.podspec]: 👍

**Pull request merged**